### PR TITLE
abdalla/swiper -> embla carousel

### PR DESCRIPTION
**Pull request by abdalla-binary**
# Description

Summary

Changes:

-   ...

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [x] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [x] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,6 +7,7 @@
         "@loadable/component": "^5.13.2",
         "commander": "^6.1.0",
         "crc-32": "^1.2.0",
+        "embla-carousel": "^4.0.5",
```
Comment (mustofa-binary) [inline package.json:10]: can we remove react-id-swiper at once

File: src/components/elements/carousel/carousel-style.js [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+    @media ${device.tabletL} {
+        display: none;
+    }
+`
+
+export const StyledChevron = styled(Chevron)`
+    height: 24px;
+    width: 24px;
+    ${(props) => (props.red ? 'width: 16px; height: 16px;' : null)}
```
Comment (mustofa-binary) [inline src/components/elements/carousel/carousel-style.js:59]: you can use css`` here

File: src/pages/about/_leaders.js [hunk 3]
```diff
@@ -313,31 +290,24 @@ const Leaders = () => {
...
-                            ))}
-                        </Swiper>
-                    </SwiperWrapper>
-                </SliderWrapper>
-            </Container>
+            <Show.Mobile>
+                <Carousel {...settings}>
+                    {leaders_data.map((leader, idx) => (
+                        <div key={idx}>
+                            <QueryImage data={data[leader.image + '_mobile']} alt={leader.name} />
```
Comment (mustofa-binary) [inline src/pages/about/_leaders.js:297]:
```suggestion
                            <QueryImage data={data[`${leader.image}_mobile`]} alt={leader.name} />
```

File: src/pages/about/_leaders.js [hunk 4]
```diff
@@ -313,31 +290,24 @@ const Leaders = () => {
...
+                            <Header size="24px" align="center" m="16px 0 0 0">
```
Comment (mustofa-binary) [inline src/pages/about/_leaders.js:298]:
```suggestion
                            <Header size="24px" align="center" m="16px 0 0">
```

**Review by mustofa-binary — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (abdalla-binary) [reply package.json:10]: We are not using it anymore so yes it is possible

**Review by mustofa-binary — APPROVED** [review]

**Review by mustofa-binary — APPROVED** [review]

**Pull request merged**